020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,194 @@
+### Dedicated cluster
+
+You can select which version you want, currently supported versions in all [available regions](/docs/getting-started/available-regions):
+
+- 15.4
+- 15.5
+- 14.10
+- 14.9
+
+You can control the maintenance windows and the version upgrades yourself.
+
+## Extensions
+
+The available, out-of-the-box extensions are:
+
+- list
+
+If you are on a dedicated cluster you can install extensions yourself, please consult the [Postgres documentation](https://www.postgresql.org/docs/current/sql-createextension.html) for a how to.
+
+## Supported Statements
+
+Xata offers different approaches to Postgres, from SQL over HTTP to pure Postgres over the wire protocol to a shared or dedicated cluster.
+Each come with a security scope, to provide isolation in a multi tenant environment.
+
+|                   | FUNCTIONS | DQL statements | (some) DML statements | DDL statements |
+| ----------------- | --------- | -------------- | --------------------- | -------------- |
+| SQL over HTTP     | N         | Y              | Y                     | Y              |
+| Shared cluster    | N         | Y              | Y                     | Y              |
+| Dedicated cluster | Y         | Y              | Y                     | Y              |
+
+The table below, is providing a more granular picture of the supported statements.
+
+|                         | SQL over HTTP | Shared Cluster | Dedicated Cluster |
+| ----------------------- | ------------- | -------------- | ----------------- |
+| `SELECT`                | Y             | Y              | Y                 |
+| `INSERT`                | Y             | Y              | Y                 |
+| `DELETE`                | Y             | Y              | Y                 |
+| `UPDATE`                | Y             | Y              | Y                 |
+| `SHOW`                  | Y             | Y              | Y                 |
+| `ALTER TABLE`           | Y             | Y              | Y                 |
+| `CREATE TABLE`          | Y             | Y              | Y                 |
+| `DROP TABLE`            | Y             | Y              | Y                 |
+| `BEGIN`                 | N             | Y              | Y                 |
+| `COMMIT`                | N             | Y              | Y                 |
+| `ROLLBACK`              | Y             | Y              | Y                 |
+| `SET`                   | Y             | Y              | Y                 |
+| `COPY`                  | Y             | Y              | Y                 |
+| `LOCK`                  | Y             | Y              | Y                 |
+| `PREPARE`               | Y             | Y              | Y                 |
+| `EXECUTE`               | Y             | Y              | Y                 |
+| `TRUNCATE`              | Y             | Y              | Y                 |
+| `ALTER TABLESPACE`      | N             | N              | Y                 |
```

Comment:
Also, I decided to remove support for tablespaces for now. It would allow users to change the path where the tables are stored. As they do not have access to the machine Postgresql is running on, it does not make sense for them to configure it.

Reply:
addressed via https://github.com/xataio/mdx-docs/pull/281/commits/b89bc30ca0617bd5b704e334161a4cf411ed9df7 - @kvch could you confirm this is correct and resolve the conversation? thanks